test/CMakeLists.txt

```diff
@@ -225,3 +225,22 @@ if (FMT_PEDANTIC AND NOT WIN32)
     "-DPEDANTIC_COMPILE_FLAGS=${PEDANTIC_COMPILE_FLAGS}"
     "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}")
 endif ()
+
+#
+# Activate CUDA related tests if we can find CUDA from CMake. This is optional.
+# For version selection, see https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#cpp14-language-features
+#
+find_package(CUDA 9.0)
+if(CUDA_FOUND)
+  add_test(cuda-test ${CMAKE_CTEST_COMMAND}
```

Comment:
What I intended was to show the full example for importing in CUDA project. But if you don't prefer that, we can consider 2 alternatives.

* `include(fmt-cuda-test.cmake)` + `add_test`: Since the CMakeLists.txt in the CUDA test folder is a bit long, we can consider `include` to shorten `test/CMakeLists.txt`
* `add_subdirectory` + `add_test`: I think this is the best